samples/attested_tls/server/host/host.cpp

```diff
@@ -43,15 +45,27 @@ int main(int argc, const char* argv[])
     oe_result_t result = OE_OK;
     int ret = 1;
     char* server_port = NULL;
+    bool keep_server_up = false;
 
     /* Check argument count */
     if (argc != 3)
     {
+        if (argc == 4)
+        {
+            if (strcmp(argv[3], LOOP_OPTION) != 0) {
+                goto print_usage;
+            }
+            else {
+                keep_server_up = true;
+                goto read_port;
+            }
+        }
     print_usage:
-        printf("Usage: %s TLS_SERVER_ENCLAVE_PATH -port:<port>\n", argv[0]);
+        printf("Usage: %s TLS_SERVER_ENCLAVE_PATH -port:<port> [-server-in-loop]\n", argv[0]);
```

Comment:
Ok as is, but also consider a precedent where the option name is passed as an argument to guarantee it's identical.
```suggestion
        printf("Usage: %s TLS_SERVER_ENCLAVE_PATH -port:<port> [%s]\n", argv[0], LOOP_OPTION);
```

Reply:
Thanks, thats a good point!